apps/system/js/accessibility.js

```diff
@@ -309,7 +325,17 @@
           return;
       }
 
-      this.speak(aDetails.data, null, {
+      var accessibilityObj = this;
+      function hintCallback() {
+        if (options.hints) {
+          if (accessibilityObj.hintTimer) {
```

Comment:
Do we really need to check for hintTimer here since we're already checking for it in Accessibility.handleEvent()?